### PR TITLE
#MAN-248 revise website header to match wireframe

### PR DESCRIPTION
**Pull request by cdoyle-temple**
#MAN-248 revise website header to match wireframe

5/2/2019 A few tweaks please:

- Can you double the lines between the navigation items? I think it should be 2px.
- Confirm the line color (About, Visit, etc) is #2a7050
- The lines between hours, chat and my account should be teal: #1a9682
- remove the line to the left of "my account"
- adjust spacing so that the lines between "my account" "hours" and "chat" have even spacing between the lines, text and icons (right now there is more space to the left of the line than to the right)
- when I shrink my browser window down, there is a gap where the navigation completely disappears before the hamburger menu reappears
- Adjust mobile navigation/search menus to Cynthia Schwarz's specs
- remove right outline from the search box (between search field and the magnifying glass icon/button)-- I would prefer the outline go around the whole thing, around the icon box too if you can (see screenshot) Screen Shot 2019-05-02 at 11.09.33 AM.png
- the temple T in the header should left align with the word "About" in the navigation
- The right edge of the search box should align with the right side of the word "hours" in the navigation

5/2/2019 Mobile Navigation Specs - as decided on 5/1:

- Main menu: please revert to what you had originally where the "Main menu" was below the logo and search icon.
- When main menu is open, hamburger converts to "X" icon.
- Search bar is within the page, not just a link to librarysearch bento
- Search bar open above the main menu
- Scrolling alert bar remains as-is in terms of location, styling and functionality
- Secondary menu appears below main menu

**********************************************************

Add search to header.
Move main nav to green bar.
Finish desktop (1200px and higher) positioning.
Update searchbox classes for tablet view positioning.
Update for responsive styling under 1200px.
Add submenu on Policies page as example for sub-pages.
Updates to breakpoints and styles
Finish layouts and positioning for mobile views
Mobile menu tweaks for discussion
Added mobile search
Adjusted mobile menus
Make tweaks noted in ticket as of 5/2/2019

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -19,3 +19,7 @@
 //= require tinymce-jquery
 //= require turbolinks
 //= require_tree .
+
+function myFunction(x) {
```
Comment (bibliotechy) [inline app/assets/javascripts/application.js]: We need to call this something other than `myFunction` that tells us what this does so we can maintain this.

File: app/views/application/_header_navbar.html.erb [hunk 2]
```diff
@@ -1,30 +1,21 @@
...
-    </ul>
+    <%= render "main-search" %>
   </div>
-  <div id="main-search" class="col-3 d-block d-md-none d-lg-none text-right">
-    <form action="https://librarysearch.temple.edu/bento">
-      <input type="submit" value = "">
-    </form>
+  <div id="mobile-search-button" class="col-3 d-block d-md-none d-lg-none text-right">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mobile-search" aria-controls="mobile-search" aria-expanded="false" aria-label="Toggle search form">
+    <span class="navbar-toggler-icon" onclick="myFunction(this)">
```
Comment (bibliotechy) [inline app/views/application/_header_navbar.html.erb]: Is `onclick` an accessibility concern? Don't keyboards nav hitting enter no respond to it?

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (cdoyle-temple) [reply app/assets/javascripts/application.js]: right

File: app/views/application/_header_navbar.html.erb (hunk 2 above)
Comment (cdoyle-temple) [reply app/views/application/_header_navbar.html.erb]: The ticket for the event listener will cover that, just need to add the enter keycode as well.

**Pull request merged**